### PR TITLE
11.0 mig account partner reconcile

### PR DESCRIPTION
**Pull request by jaumeplanas**

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Next time, you don't need to close the other PR and reopen this. You only need to overwrite the remote branch using the same branch name.

**Comment by pedrobaeza** [comment]
@jbeficent you were the original author of this. Shouldn't it be in OCA/account-reconcile?

File: account_partner_reconcile/views/res_partner_view.xml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record id="res_partner_view_buttons" model="ir.ui.view">
+        <field name="name">res.partner.view.buttons</field>
+        <field name="model">res.partner</field>
+        <field name="inherit_id" ref="base.view_partner_form" />
+        <field name="priority" eval="20"/>
+        <field name="groups_id" eval="[(4, ref('account.group_account_user'))]"/>
```
Comment (JordiBForgeFlow) [inline account_partner_reconcile/views/res_partner_view.xml:9]: Puedes cambiar el grupo por 'account.group_account_invoice'.

el grupo 'group_account_user' es un rollo porque requiere que se marque en el usuario 'show full accounting features' lo cual no es en absoluto necesario en este caso.

Los usuarios en el grupo 'group_account_invoice' ya entrar cobros y conciliarlos con facturas desde allí.

**Review by JordiBForgeFlow — CHANGES_REQUESTED** [review]
cambiar grupo

**Comment by JordiBForgeFlow** [comment]
Closing in favor of https://github.com/OCA/account-payment/pull/187

**Pull request closed without merging**